.vscode/tasks.json

```diff
@@ -1,33 +1,13 @@
 {
 	"version": "2.0.0",
 	"tasks": [
-		{
-			"type": "func",
-			"label": "func: host start",
-			"command": "host start",
-			"problemMatcher": "$func-python-watch",
-			"isBackground": true,
-			"dependsOn": "pip install (functions)",
-			"options": {
-				"cwd": "${workspaceFolder}/code/backend/batch"
-			}
-		},
-		{
-			"label": "pip install (functions)",
-			"type": "shell",
-			"command": "python -m pip install -r requirements.txt",
-			"problemMatcher": [],
-			"options": {
-				"cwd": "${workspaceFolder}/code/backend"
-			}
-		},
 		{
 			"label": "pip install (code)",
 			"type": "shell",
-			"command": "python -m pip install -r requirements.txt",
+			"command": "pip install .",
```

Comment:
Should this be `poetry install` instead now? Might need to rename the label. Maybe it should just be generic like `install dependencies`?

Reply:
Note that any name changes to your tasks need to be reflected in the launch.json that call these tasks. e.g. `launch Admin site` has a `preLaunchTask` that it cannot find now